lib/canvas.rb

```diff
@@ -0,0 +1,4 @@
+# ...the game board
+class Canvas
+  attr_accessor :array_canvas
```

Comment:
* [ ] Use attr_readers/attr_writers instead of attr_accessors. Using attr_accessors allows your property to be overwritten from outside of the class.  That variable can be set to anything. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))